.gitignore

```diff
@@ -1,6 +1,7 @@
 *.userprefs
 bin
 Configuration.Override.props
+Configuration.OperatingSystem.props
```

Comment:
I think you are overthinking it slightly :) The file contains information pertaining to the operating system, hence I thought it was the most logical name